src/docs/deprecation_policy.md

```diff
@@ -3,7 +3,7 @@ Pants Deprecation Policy
 
 Deprecations should live for one entire `dev` series and stable release branch (at the minimum) before their code is removed. This ensures that there is always at least one stable release containing a deprecation, and allows users upgrading their copy of pants to consume only stable releases in order to observe all deprecations.
 
-As an example: if a feature is deprecated during the `1.0.0.dev` series, it should remain available but deprecated for the entire `1.1.0.dev` series, and can be removed when master bumps to `1.2.0.dev0`.
+As an example: if a feature is deprecated in `1.0.0.dev2`, it should remain available but deprecated for the entire `1.1.0.dev` series, and can be removed when master bumps to `1.2.0.dev0`.
```

Comment:
Do we want to go ahead and make `dev1` explicit here?

Reply:
Ah, you meant over Slack that we want to always target `dev1` rather than `dev0`? I misunderstood that you only meant in terms of bumping up deprecations in this PR.

I'd strongly prefer that we always target `dev0`. The risk of harder cherry-picks for the benefit of faster iteration speed seems worth it to me, particularly because we want to encourage an incremental migration for V2. We don't need to cherry-pick very frequently, and when we do, to me it seems acceptable to have some merge conflicts that need to be resolved.

But I realize the perspective I'm coming at this from. I've never had to release an RC and I'm the main person making all these deprecations recently, so talk is cheap for me talking about merge conflicts when cherry-picking.

Would you feel better about `dev0` if we started asking the person who made the commit that needs to be cherry-picked to do the cherry-pick themself? It seems that the burden usually falls on the releaser.

Reply:
I'm fine with trying it out. I think that post `2.0.0` we should discuss getting more of the community involved in the stable release process so that the burden of stability falls on more than just Twitter, and we're able to discuss these kinds of costs from the same context.

> Would you feel better about dev0 if we started asking the person who made the commit that needs to be cherry-picked to do the cherry-pick themself? It seems that the burden usually falls on the releaser.

When the pick is challenging, this generally happens already in a more adhoc fashion.